src/tools/qt-gui/qml/AboutWindow.qml

```diff
@@ -62,21 +62,500 @@ BasicWindow {
 						property string pancheri: "mailto:e0003088@student.tuwien.ac.at"
 						property string loose: "mailto:christian.loose@hamburg.de"
 						property string jens: "http://qt-project.org/forums/viewthread/30521/#146845"
+						property string ian: "mailto:easyxtarget@gmail.com"
 						readOnly: true
 						textFormat: TextEdit.RichText
 						textMargin: defaultSpacing
 						wrapMode: Text.WordWrap
 						text: "<html>" +
 							  "<style type=\"text/css\"></style>" +
-							  "<p>Elektra Editor created by <a href=\"" + pancheri + "\">Raffael Pancheri</a></p>" +
+							  "<p>Elektra Editor designed and implemented by <a href=\"" + pancheri + "\">Raffael Pancheri</a></p>" +
 							  "<p>DiscountMarkdownConverter used with kind permission by <a href=\"" + loose + "\">Christian Loose</a></p>" +
 							  "<p>TreeView based on code <a href=\"" + jens + "\">posted by Jens</a></p>" +
+							  "<p>Helptext by <a href=\"" + ian + "\">Ian Donelly</a></p>" +
 							  "</html>"
 						onLinkActivated: {
 							Qt.openUrlExternally(link)
 						}
 					}
 				}
+				Tab {
+					id: licenseTab
+
+					title: qsTr("License")
+					TextArea {
```

Comment:
I dont think we need creative commons full text here (especially not this long variant). Just state that the icons are used and put a link to the licence.